theme/snippets/news-feed.liquid

```diff
@@ -1,13 +1,14 @@
 <div class="NewsFeed flex flex-col">
-  <div class="NewsFeed__inner-container overflow-x-scroll flex flex-col border-top-black border-bottom-black">
-  {% for value in shop.metafields.accentuate.news_feed_item_body limit:4 %} 
+  <div class="NewsFeed__inner-container overflow-x-scroll flex flex-col border-thick-top-black border-thick-bottom-black">
+    {% for value in shop.metafields.accentuate.news_feed_item_body limit: limit %} 
     {% assign multi_thumbnails = shop.metafields.accentuate.news_feed_item_thumbnail[forloop.index0] %}
     {% for thumbnail in multi_thumbnails %} 
       {% assign current_thumbnail = thumbnail %}
     {% endfor %}
     {% render 'news-feed-item', image_src: current_thumbnail.src image_alt: current_thumbnail.alt body: value, third_column: shop.metafields.accentuate.news_feed_item_third_column[forloop.index0] %}
-  {% endfor %} 
+  {% endfor %}
   </div>
-
-  <a class="uppercase serif-xl pt2 md:pt5 pb3 hover-lighten-black" href="/pages/news" aria-label="{{ 'snippets.news_feed.see_more_link.aria_label' | t }}">{{ 'snippets.news_feed.see_more_link.text' | t }}</a>
+  {% if is_limited %}
```

Comment:
Can we just use some liquid logic to check if limit and limit > 0 instead of having another property that we need to manually set to determine if the 'see more' button shows or not?